cdtb/storage.py

```diff
@@ -11,6 +11,7 @@
 
 from .tools import write_file_or_remove
 
+
```

Comment:
was this intended?

Reply:
Hum... maybe not. Other files only have 1 empty line before logger import, I'll remove it.